README.md

```diff
@@ -99,6 +99,20 @@ export function App() {
 }
 ```
 
+### onChangeComplete optional callback
+```tsx
+import { ColorPicker, useColor } from "react-color-palette";
```

Comment:
Let's add the `IColor` interface to the imports to make it more explicit for the rest of the code.

```tsx
import { ColorPicker, useColor, type IColor } from "react-color-palette";
```

Reply:
```suggestion
import { ColorPicker, useColor, type IColor } from "react-color-palette";
```